packages/drivers/odsp-driver/src/fetchSnapshot.ts

```diff
@@ -313,6 +314,7 @@ async function fetchLatestSnapshotCore(
                     encodedBlobsSize,
                     sequenceNumber,
                     ops: snapshot.ops?.length ?? 0,
+                    nonSysOps:  snapshot.ops?.filter((op) => !isSystemMessage(op)).length ?? 0,
```

Comment:
Instead of 0, should it undefined? Since 0 could be misleading.

Reply:
with Ops we set to 0, so I was assuming similar behavior.

Reply:
We usually prefer undefined, but here it depends on if there is difference on spo side between no array or empty array (and if both are used for different flows).
Given that we do not know if such diff exists, it’s ok as is